projects/igniteui-angular/src/lib/directives/mask/mask.directive.ts

```diff
@@ -183,6 +185,18 @@ export class IgxMaskDirective implements OnInit, AfterViewChecked, ControlValueA
          * the end user will be unable to blur the input.
          * https://stackoverflow.com/questions/21406138/input-event-triggered-on-internet-explorer-when-placeholder-changed
          */
+        if (isComposing) {
```

Comment:
Unfortunately, it seems `inputEvent.isComposing` is experimental still and not really available in IE11 and more importantly in Safari per https://developer.mozilla.org/en-US/docs/Web/API/InputEvent.
Considering it's basically a flag if the event is fired after compositionstart and before compositionend, perhaps it'd be more reliable if we handle that ourselves?